sentry/src/main/java/io/sentry/SentryClient.java

```diff
@@ -317,15 +343,26 @@ public void setExtraTags(Set<String> extraTags) {
     }
 
     /**
-     * Add an extra to extract and send as tags on all future {@link Event}s, where applicable.
-     * <p>
-     * For example: when using a logging integration any {@link org.slf4j.MDC} keys that are in
-     * the {@link #extraTags} set will be extracted and set as tags on the {@link Event}.
+     * Add a extra data that will be sent with all future {@link Event}s.
```

Comment:
an extra

"an extra data" is probably the wrong noun to use here anyway, probably better as one of "an extra data {item, value, member}"